jest.config.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
   snapshotSerializers: ['<rootDir>/node_modules/jest-serializer-vue'],
 
   // Coverage
+  coverageProvider:    'v8',
```

Comment:
Without this we can't run coverage and have the two `<script>` tags in one file. Without this jest will complain about having a duplicate identifier.